app/views/categories/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Categories</h1>
+
+<div class="btn-group btn-group-lg">
+  <button type="button" class="btn btn-primary"><a href="http://localhost:3000/categories/1/items">Mac</a></button>
+  <button type="button" class="btn btn-primary"><a href="http://localhost:3000/categories/2/items">Iphone</a></button>
+  <button type="button" class="btn btn-primary"><a href="http://localhost:3000/categories/3/items">Accessories</a></button>
```

Comment:
These categories should be loaded from DB - that is why we agreed to prepare so called "seeds" . Below are some example solutions:

Iterate through all categories:

```ruby
Category.all do |category|
  ...
end
```

define methods that will refer to appropriate records:

```
Category.mac
Category.iphone
Category.accessories
```